packages/celotool/src/cmds/generate/prepare-load-test-client.ts

```diff
@@ -0,0 +1,51 @@
+/* tslint:disable no-console */
+import * as fs from 'fs'
+import { AccountType, generatePrivateKey, privateKeyToAddress } from 'src/lib/generate_utils'
+import yargs from 'yargs'
+const keythereum = require('keythereum')
+
+interface Bip32Argv {
+  mnemonic: string
+  index: number
+  threads: number
+}
+
+export const command = 'prepare-load-test'
+
+export const describe = 'command for generating public and private keys for a load test instance'
+
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .option('mnemonic', {
+      type: 'string',
+      description: 'BIP-39 mnemonic',
+      demandOption: 'Please specify a mnemonic from which to derive a private key',
+      alias: 'm',
+    })
+    .option('index', {
+      type: 'number',
+      description: 'Index of key to generate',
+      demandOption: 'Please specify a key index',
+      alias: 'i',
+    })
+    .option('threads', {
+      type: 'number',
+      description: 'The number of threads',
+      demandOption: 'Please specify the number of threads of this node',
+      alias: 't',
+    })
+}
+
+export const handler = async (argv: Bip32Argv) => {
+  const accountType = AccountType.LOAD_TESTING_ACCOUNT
+  for (let t = 0; t < argv.threads; t++) {
+    const index = argv.index * 10000 + t
+
+    const privateKey = generatePrivateKey(argv.mnemonic, accountType, index)
+    const address = privateKeyToAddress(privateKey)
+    fs.writeFileSync(`/root/.celo/pkey${t}`, `${privateKey}\n`)
```

Comment:
Is it ok to try to write to `/root` from a celotool command?  I don't see that being done in other commands (though I may have missed it).  Is the usual pattern to do the writing from the helm chart instead?

Reply:
I'd say it is. The `prepare-load-test-client` is intended to run only in a container inside the load-test pod, to prepare all the private keys, and address file needed later by geth and the load-test client itself. The only shared folder between containers inside the pod is `/root/.celo`. We could write to another folder and then in the pod script move to `/root/.celo/` but seems a bit artificial to me.